front/src/applications/operationalStudies/components/SimulationResults/ChartModal/ChartModal.js

```diff
@@ -20,13 +18,15 @@ function ChartModal(props) {
       setShowModal('');
       const seconds = parseInt(type === '-' ? offset * -1 : offset, 10);
       offsetTimeByDragging(seconds);
+      /*
       changeTrain(
         {
           departure_time: simulation.trains[selectedTrain].base.stops[0].time + seconds,
         },
         simulation.trains[selectedTrain].id
       );
       dispatch(updateMustRedraw(true));
+      */
```

Comment:
delete commented code (you can always find it in the PR changes)

Reply:
Done